handlers/ticket_test.go

```diff
@@ -247,58 +247,29 @@ func TestUpdateTicket(t *testing.T) {
 		assert.Equal(t, http.StatusBadRequest, rr.Code)
 	})
 
-	t.Run("should update ticket with only UUID and optional fields", func(t *testing.T) {
+	t.Run("should update ticket with new format", func(t *testing.T) {
 		rr := httptest.NewRecorder()
 		handler := http.HandlerFunc(tHandler.UpdateTicket)
 
-		// Create a ticket with only UUID and some optional fields
-		updateTicket := db.Tickets{
-			UUID:        createdTicket.UUID,
-			Description: "Updated description", // Optional field
-			Status:      db.ReadyTicket,        // Optional field
+		updateRequest := UpdateTicketRequest{
+			Metadata: struct {
+				Source string `json:"source"`
+				ID     string `json:"id"`
+			}{
+				Source: "websocket",
+				ID:     "ws-12345",
+			},
+			Ticket: &db.Tickets{
+				UUID:        createdTicket.UUID,
+				Description: "Updated Description via new format",
+				Status:      db.ReadyTicket,
+			},
 		}
 
-		requestBody, _ := json.Marshal(updateTicket)
```

Comment:
Can you explain the deletion here in the test? 

Can this up dated instead of deleted?